src/test/java/core/basesyntax/FruitServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax;
+
+import static core.basesyntax.service.impl.Operation.BALANCE;
+import static core.basesyntax.service.impl.Operation.PURCHASE;
+import static core.basesyntax.service.impl.Operation.SUPPLY;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.AddOperation;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.Operation;
+import core.basesyntax.service.impl.SubtractOperation;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static Map<Operation, FruitOperationHandler> operations = new HashMap<>();
+    private static FruitService fruitService;
+
+    @BeforeClass
+    public static void operationsInitialization() {
+        operations.put(BALANCE, new AddOperation());
+        operations.put(Operation.PURCHASE, new SubtractOperation());
+        operations.put(Operation.RETURN, new AddOperation());
+        operations.put(Operation.SUPPLY, new AddOperation());
+    }
+
+    @BeforeClass
+    public static void initialization() {
```

Comment:
what's the difference between these two methods?

Reply:
United these in one method.